### DISCUSSION TITLE
MFA user authentication

### DISCUSSION THREAD

**Discussion by jianiwa** [pnp/pnpframework#374] [Q&A]
Is there an example on how to connect the AuthManager when the user requires MFA?
I am getting the below error for this line:
`var authManager = new AuthenticationManager(clientId, userName, securePassword);`

```
Microsoft.Identity.Client.MsalUiRequiredException: 'AADSTS50076: Due to a configuration change made by your administrator, or because you moved to a new location, you must use multi-factor authentication to access {guid}
```

**Comment by jansenbe** [comment]
You should use an interactive login flow that will open the browser to let the user authenticate using MFA. See the `CreateWithInteractiveLogin` options in the `AuthenticationManager` class
